_rules/SC1-1-1-accessible-name-is-image-filename.md

```diff
@@ -0,0 +1,95 @@
+---
+name: Image accessible name is filename
+description: |
+  This rule checks that image elements that use their source filename as their accessible name do so without loss of information to the user.
+
+success_criterion:
+- 1.1.1 # Non-Text Content
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Bryn Anderson
+---
+
+## Test procedure
+
+### Applicability
+
+The rule applies to any HTML `input` element with a [`type`](https://www.w3.org/TR/html/sec-forms.html#dom-htmlinputelement-type) of `image`, or any HTML `area` element, or any HTML element with the [semantic role](#semantic-role) of `img`. Applicable elements are [non-decorative](#decorative), [included in the accessibility tree](#included-in-the-accessibility-tree), and have an [accessible name](#accessible-name) that is equivalent to the [filename](#filename) specified in the `src` attribute. Difference in letter casing, and forward and trailing whitespace should be ignored.
```

Comment:
This seems to imply taking the `src` attribute from an `area` element. The `area` element does not have an `src` attribute, so that doesn't make sense. I think this rule should ignore `area` elements for that reason.

Reply:
removed `area` elements